glowbase/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:14.04
+
+MAINTAINER Andrea Grandi <andrea@thisisglow.com>
+
+RUN apt-get update -y && \
+apt-get install -y python-dev build-essential python-pip libcurl4-gnutls-dev python2.7-dev libpq-dev supervisor nfs-common git nginx libjpeg-dev npm nodejs-legacy python-dev build-essential python-pip python2.7-dev wget unzip openjdk-6-jre git ruby nginx && \
```

Comment:
Not sure if all these packages should be there... expecially JDK :)
Maybe we optimize for speed but not for size in this way.